test/Microsoft.Azure.Devices.Edge.Test.Common/linux/EdgeDaemon.cs

```diff
@@ -35,6 +35,7 @@ public class EdgeDaemon : IEdgeDaemon
             switch (os)
             {
                 case "Ubuntu":
+                case "NXP-LSDK":
```

Comment:
Since we want to check in generic support for ARM64, let's remove this board-specific reference for now.

Reply:
Removed